tests/functional/cssTranstions.ts

```diff
@@ -0,0 +1,62 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import pollUntil = require('intern/dojo/node!leadfoot/helpers/pollUntil');
+
+registerSuite({
+	name: 'cssTransitions',
+
+	'-active gets appended to enter/exit animations by default'(this: any) {
```

Comment:
Maybe? Can we unit test changes to DOM elements? I see we have JSDOM loaded in widget-core, but I'm not really sure what I can do with that.